networking/src/main/java/ar/com/wolox/wolmo/networking/utils/LoggingUtils.java

```diff
@@ -19,41 +19,40 @@
  * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
  * DEALINGS IN THE SOFTWARE.
  */
+package ar.com.wolox.wolmo.networking.utils;
 
-package ar.com.wolox.wolmo.networking.retrofit;
-
-import android.support.annotation.CallSuper;
-
-import ar.com.wolox.wolmo.core.WolmoApplication;
+import android.support.annotation.NonNull;
 
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
 
 /**
- * This class extends {@link ar.com.wolox.wolmo.core.WolmoApplication} to easily initialize and get and instance of
- * {@link RetrofitServices} to perform API calls.
+ * Utils to create {@link HttpLoggingInterceptor} for {@link OkHttpClient}.
  */
-public abstract class NetworkingApplication extends WolmoApplication {
+public class LoggingUtils {
```

Comment:
Should we build a class just to return an `HttpLoggingInterceptor`?

If it provided several different `Interceptor`s or something like that, i would be more in favor. But i believe this code can be inlined.

Reply:
I think I'll remove this class and let the client to create and configure a logger interceptor if they want. Do you think we should keep the library `com.squareup.okhttp3:logging-interceptor`?